unit_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cd $APP_ROOT
+
+# Deploy ephemeral DB
+source $CICD_ROOT/deploy_ephemeral_db.sh
+
+# Map env vars set by `deploy_ephemeral_db.sh` if vars the app users are different
```

Comment:
Along with the DB will we also be needing to provision a redis instance?

Reply:
I don't think your tests require redis as far as I can tell.